src/orange/OrangeParams.cc

```diff
@@ -180,15 +178,63 @@ OrangeParams::OrangeParams(OrangeInput input)
             }
             volume_labels.push_back(std::move(volume_label));
         }
-
-        bbox_ = u.bbox;
     }
 
     surf_labels_ = LabelIdMultiMap<SurfaceId>{std::move(surface_labels)};
     vol_labels_ = LabelIdMultiMap<VolumeId>{std::move(volume_labels)};
+}
```

Comment:
For volume labels I can use the same convention as is written to json, in the format "(i, j, k)". For surface labels, I think we need labels for Nx + Ny + Nz surfaces, whereas json only names the outer surfaces. So we need a new convention. How about "(ax, index)", e.g. "(z, 22)"

Reply:
Actually, we probably don't want to read them from JSON. Then the Geometria Celeritas_Converter, when converting from SCALE to CELERITAS ORANGE objects, would have to provide each volume with a label, which seems unnecessary. I will just have it so that RectArrayInserter creates the volume and surface labels.

Reply:
@sethrj thinking about this more, I don't actually think we need volume labels. In an array universe, volumes have local ids, but not global ids. A particle's global volume id always corresponds to a cell in a daughter universe.

Surfaces on the other hand needs to have global surface ids. Likewise, I need to account for this in UnitIndexer. I am fixing this now.